sdks/wasm/netcore-corefx.diff

```diff
@@ -0,0 +1,55 @@
+diff --git a/src/Native/Unix/CMakeLists.txt b/src/Native/Unix/CMakeLists.txt
```

Comment:
Why is this needed? We should not be building this under Mono

Reply:
There are no wasm binaries for System.Native.

Reply:
We should bring them back and publish through NuGet. That is no trivial effort though. At some point the WASM targets existed in CoreFX but they were removed and never made it to Azure Pipelines. I've been told that there should be a machine pool with emscripten and other relevant tools installed but I know no details about that.

Reply:
I agree we need to start the effort on corefx side as well (I'll look into that)

Reply:
For reference, the initial WebAssembly support was added with https://github.com/dotnet/corefx/pull/28936. Various parts were removed by https://github.com/dotnet/corefx/pull/36739.

Reply:
Here's the definition for the Azure Pipelines pool:
```
        pool:
            name:  NetCorePublic-Pool
            queue: buildpool.windows.10.wasm.open
```

Reply:
> Various parts were removed by dotnet/corefx#36739

The initial WebAssembly support was on top of ProjectN. The removed parts were the ProjectN specific plumbing.